README.md

```diff
@@ -40,15 +41,15 @@ Please check **Required parameters** column to see if you need to set any additi
 | [huggingface_llm](/label_studio_ml/examples/huggingface_llm)               | Text generation models by [Hugging Face](https://huggingface.co/tasks/text-generation)                                                           | None                |
 | [huggingface_ner](/label_studio_ml/examples/huggingface_ner)               | NER models (TokenClassification) by [Hugging Face](https://huggingface.co/docs/transformers/en/tasks/token_classification)                       | None                |
 | [nemo](/label_studio_ml/examples/nemo)                                     | Speech transcription models by [NVIDIA NeMo](https://github.com/NVIDIA/NeMo)                                                                     | None                |
-| [mmetection](/label_studio_ml/examples/mmetection)                         | Object detection models by [OpenMMLab](https://github.com/open-mmlab/mmdetection)                                                                | None                |
+| [mmdetection](/label_studio_ml/examples/mmdetection-3)                         | Object detection models by [OpenMMLab](https://github.com/open-mmlab/mmdetection)                                                                | None                |
```

Comment:
@niklub - the link was broken, so I wasn't sure which example you wanted to use. I linked to this one because it had been updated more recently